commands/user_e2e_test.go

```diff
@@ -136,3 +136,46 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUserConvertCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Error when no flag provided", func(c client.Client) {
+		printer.Clean()
+
+		emailArg := "example@example.com"
+		cmd := &cobra.Command{}
+
+		err := userConvertCmdF(c, cmd, []string{emailArg})
+		s.Require().Error(err)
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Valid user to bot convert", func() {
+		printer.Clean()
+
+		email := s.th.BasicUser.Email
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("bot", true, "")
+
+		err := userConvertCmdF(s.th.SystemAdminClient, cmd, []string{email})
+		s.Require().NoError(err)
+		s.Require().Len(printer.GetLines(), 1)
```

Comment:
Shall we check if we have the expected output here?

Reply:
Done.